tests/Integration/HttpClientTest.php

```diff
@@ -20,8 +21,8 @@ public function testHttpClient()
 
     public function testHttpClientInvalidTimeout()
     {
-        $this->expectException(\InvalidArgumentException::class);
-        $this->expectExceptionMessageRegExp('/^Timeout must be an int > 0, got "integer 0".$/');
+        $this->expectException(InvalidArgumentException::class);
+        $this->expectExceptionMessage('Timeout must be an int > 0, got "integer 0"');
```

Comment:
`expectExceptionMessageRegExp` can be safely replaced by `expectExceptionMessage` as per the `phpunit/phpunit` package upgrade.

Reply:
And this actually also enforces a stricter assertion 🚀